src/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+
+function name(fullInfo) {
```

Comment:
It's really great that you've extracted this component, but there's a few of problems:

1. Do you think `name` describes what this component is doing very well? What do you think a better name would be?
2. In React, we normally like to start component names with a capital letter
3. We also like to put each component in it's own file so it's clear where to look when you want to change that component. Could you move this to it's own file?